sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/jsonExpressions.scala

```diff
@@ -509,9 +509,9 @@ case class JsonTuple(children: Seq[Expression])
   examples = """
     Examples:
       > SELECT _FUNC_('{"a":1, "b":0.8}', 'a INT, b DOUBLE');
-       {"a":1,"b":0.8}
+       {1, 0.8}
```

Comment:
The doc example of `CreateNamedStruct` is
```
> SELECT _FUNC_("a", 1, "b", 2, "c", 3);
       {"a":1,"b":2,"c":3}
```

I think it's a valid syntax for named rows.

Reply:
> The doc example of `CreateNamedStruct` is
> 
> ```
> > SELECT _FUNC_("a", 1, "b", 2, "c", 3);
>        {"a":1,"b":2,"c":3}
> ```
> 
> I think it's a valid syntax for named rows.

I got it, seems because I run this  SQL in normal UT, i just return struct data's value part. I was misled.